proxy-node-shared/src/main/java/uk/gov/ida/notification/contracts/HubResponseTranslatorDto.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.ida.notification.contracts;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.opensaml.security.x509.X509Credential;
+
+import javax.validation.constraints.NotNull;
+
+public class HubResponseTranslatorDto {
+
+    @NotNull
+    @JsonProperty
+    private String samlResponse;
+
+    @NotNull
+    @JsonProperty
+    private String requestId;
+
+    @JsonProperty
+    private String eidasRequestId;
```

Comment:
Are `eidasRequestId` and `connectorEncryptionCredential` nullable i.e. no @NotNull annotation?

Reply:
Confused by the requestId and eidasRequestId? Also shouldn't destination be included in this DTO?

Reply:
I think gateway could store destination in session, its used to build the form `action` in the  eidas saml response form.

Reply:
And then pass it on to the Translator? The destination is part of the eidas response, so I reckon we should use it in the Translator when it generates the eidas response.

Reply:
Refactored into two objects